src/sidebar/components/ShareDialog/ExportAnnotations.tsx

```diff
@@ -176,6 +176,7 @@ function ExportAnnotations({
               groupName: group?.name,
               defaultAuthority,
               displayNamesEnabled,
+              separator: context === 'file' ? ',' : '\t',
```

Comment:
Can you add a short comment here explaining why we're doing this.

Reply:
Yeah, good point.